eql/track.json

```diff
@@ -53,21 +56,25 @@
         "ingest-percentage": {{ingest_percentage | default(100)}}
       },
       "clients": {{bulk_indexing_clients | default(8)}},
-      "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
+      "ignore-response-error-level": "{{error_level | default('non-fatal')}}",
+      "tags": "setup"
```

Comment:
turns out tags are pretty helpful when working with rally, I've also updated Marios document to explain how they can be used for local testing (https://docs.google.com/document/d/1RiG-Itu532RDHQMq6FSt4iEXE0_42K15aayuYDBGk40/edit#heading=h.3um5dshkzbne)